src/coreclr/jit/block.h

```diff
@@ -640,11 +644,8 @@ struct BasicBlock : private LIR::Range
     void SetJumpKindAndTarget(BBjumpKinds jumpKind, BasicBlock* jumpDest DEBUG_ARG(Compiler* compiler))
```

Comment:
Hmm, odd that the `DEBUG_ARG` shows up here since it was removed. I guess it will fix itself when you push a new merge commit.

Reply:
Yeah, those follow-up PRs I created caused a lot of merge conflicts that I plan to clean up today, so this should disappear.